src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-add-tariff-service-pop-up/ubs-admin-tariffs-add-tariff-service-pop-up.component.ts

```diff
@@ -44,10 +47,16 @@ export class UbsAdminTariffsAddTariffServicePopUpComponent implements OnInit {
     });
     this.initForm();
     this.fillFields();
-    this.localeStorageService.languageBehaviourSubject.pipe(takeUntil(this.unsubscribe)).subscribe((lang: string) => {
-      const datePipe = new DatePipe(lang);
-      this.newDate = datePipe.transform(new Date(), 'MMM dd, yyyy');
-    });
+    this.setDate();
+  }
+
+  setDate(): void {
+    const lang = this.languageService.getCurrentLanguage();
+    this.datePipe = new DatePipe(lang);
+    this.newDate = this.datePipe.transform(new Date(), 'MMM dd, yyyy');
+    if (lang === 'en') {
+      this.isLangEn = true;
+    }
```

Comment:
the same.   setDate - duplicate method, move it to the service

Reply:
Thank you. Fixes was added. Created method in to the service.